src/python/deepgnn/graph_engine/snark/convert.py

```diff
@@ -51,9 +49,10 @@ def output(
     suffix: int,
     node_type_num: int,
     edge_type_num: int,
-    decoder_type: DecoderType,
+    decoder_class: Decoder,
```

Comment:
`decoder_class: Decoder` means you expect an object of type `Decoder`, not a class type.